bdk_core/src/lib.rs

```diff
@@ -37,14 +37,14 @@ pub struct BlockTime {
     pub time: u64,
 }
 
-/// A Blockhash and Blockheight denoting a Checkpoint in the Blockchain.
+/// A reference to a CheckPoint by specific Height and Hash.
 #[derive(Debug, Clone, PartialEq, Eq, Default, Copy)]
 #[cfg_attr(
     feature = "serde",
     derive(serde::Deserialize, serde::Serialize),
     serde(crate = "serde_crate")
 )]
-pub struct CheckPoint {
+pub struct CheckPointRef {
```

Comment:
Could we call this a `BlockId` maybe? Then "CheckPointData" just becomes `CheckPoint` I guess.

Reply:
Yup.. That sounds perfect..